lib/rules/no-duplicate-at-import-rules/index.js

```diff
@@ -73,6 +70,51 @@ const rule = (primary) => {
 	};
 };
 
+/**
+ * List the import conditions found in the prelude of an `@import` rule
+ *
+ * @param {import('postcss-value-parser').Node[]} params
```

Comment:
minor

since you are using `import('postcss-value-parser').Node` 3 times, you could create a custom type before
l76 l82 l84